config/porter.json

```diff
@@ -0,0 +1,20 @@
+{
+  "targets": [
+    {
+      "name": "orderly_volume",
+      "type": "volume"
+    }
+  ],
+  "hosts": [
+    {
+      "name": "annex",
+      "hostname": "annex.montagu.dide.ic.ac.uk",
+      "user": "montagu"
+    },
+    {
+      "name": "production",
+      "hostname": "production.montagu.dide.ic.ac.uk",
+      "port": 10022
+    }
+  ]
```

Comment:
I've left "type" in for targets in case we need plain folders, yeah. And yes, hosts should also have paths (this is in the next PR, but could add it here). Hosts will also get a "type" field so that we can have local as well as remote locations. Again this is in next PR but could add it here to make that one a bit smaller. Will do that.